applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_FIC_element.cpp

```diff
@@ -34,14 +34,14 @@ Element::Pointer UPwSmallStrainFICElement<TDim,TNumNodes>::Create(IndexType NewI
 //----------------------------------------------------------------------------------------
 
 template< unsigned int TDim, unsigned int TNumNodes >
-void UPwSmallStrainFICElement<TDim,TNumNodes>::Initialize(const ProcessInfo& rCurrentProcessInfo)
+void UPwSmallStrainFICElement<TDim,TNumNodes>::
+    Initialize(const ProcessInfo& rCurrentProcessInfo)
 {
     KRATOS_TRY
 
     UPwBaseElement<TDim,TNumNodes>::Initialize(rCurrentProcessInfo);
 
-    unsigned int VoigtSize = VOIGT_SIZE_3D;
-    if (TDim == 2) VoigtSize = VOIGT_SIZE_2D_PLANE_STRESS;
+    const SizeType VoigtSize  = ( TDim == 3 ? VOIGT_SIZE_3D : VOIGT_SIZE_2D_PLANE_STRAIN);
```

Comment:
```suggestion
    constexpr SizeType VoigtSize  = TDim == 3 ? VOIGT_SIZE_3D : VOIGT_SIZE_2D_PLANE_STRAIN;
```
This allows to create bounded matrices with `VoigtSize`. Indeed, you can specify this `constexpr` once in the begining of the class to use it al throughout the file.